src/help.cpp

```diff
@@ -367,6 +367,15 @@ void add_options() {
   opt->setSQL();
   opt->setDDL();
 
+  /*Discard tablespace */
+  opt = newOption(Option::INT, Option::ALTER_DISCARD_TABLESPACE,
+		  "alt-discard-tbs");
+  opt->help = "Alter table to discard file-per-tablespace";
+  /* Disabled by default, until https://jira.percona.com/browse/PS-7865 is fixed */
+  opt->setInt(0);
```

Comment:
disable it only version lower than 8.0.26. Hope they will fix it 8.0.27

Reply:
Fixed